app/api/v2/resource/users.rb

```diff
@@ -34,7 +34,7 @@ def validate_topic!(topic)
         end
         get '/activity/:topic' do
           validate_topic!(params[:topic])
-          data = current_user.activities
+          data = current_user.activities.order('created_at DESC')
```

Comment:
Can you also please add a spec on it?
Something simple, just `it returns activities from new to old`

Reply:
i'm poor rspec :|